src/app/view/components/_common/dot-icon/dot-icon.component.scss

```diff
@@ -17,7 +17,7 @@
     }
 
     i {
-        font-size: $md-icon-size-normal;
+        font-size: 1.571428571rem; // 22px
```

Comment:
global variable?

Reply:
We can't because is a very specific number to match 22px

Reply:
@fmontes Round it to 1.58rem